client/my-sites/plans/current-plan/header.jsx

```diff
@@ -3,29 +3,64 @@
  */
 import React from 'react';
 import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import PlanIcon from 'components/plans/plan-icon';
 import HappinessSupport from 'components/happiness-support';
+import Button from 'components/button';
+import Card from 'components/card';
```

Comment:
Nit question: should we order imported modules alphabetically? I see some do it and others not. Is it a preferred thing to do?